mix/QVariableDefinition.cpp

```diff
@@ -53,3 +55,92 @@ QVariableDefinition* QVariableDefinitionList::val(int _idx)
 		return nullptr;
 	return m_def.at(_idx);
 }
+
+/*
+ * QIntType
+ */
+void QIntType::setValue(dev::bigint _value)
+{
+	m_bigIntvalue = _value;
+	std::stringstream str;
+	str << std::dec << m_bigIntvalue;
+	m_value = QString::fromStdString(str.str());
+}
+
+dev::bytes QIntType::encodeValue()
+{
+	dev::bigint i(value().toStdString());
```

Comment:
At this point, QIntType comes from QML, so we only have the string value. 
Maybe we can add a function to resolve the bigint value in QML side for thoses specific types (bool, int, uint).